core-blocks/file/index.js

```diff
@@ -217,7 +217,7 @@ export const settings = {
 					<a
 						href={ href }
 						className="wp-block-file__button"
-						download={ fileName }>
+						download={ fileName || '' }>
```

Comment:
Why is this needed?

Reply:
I had an issue before whereby if the fileName is undefined, the download attribute is not added to the resulting anchor, which changes the behaviour of the button from downloading a file to opening it in a new tab. It's probably not needed now. Not sure if there could ever be an edge-case where a fileName could be undefined now that the string format is being used for the rich text field.

Reply:
Fair enough. I wonder if it should be `fileName || true` so that if there is no file name, the attribute is added but not set to anything? e.g. `<a href="..." download>`